### PR TITLE
Added compound and save add-on

### PR DESCRIPTION
**Pull request by sidlavanet**

### CODE REVIEW

File: cookbook/specs/spec_add_sqdsubgraph.json [hunk 1]
```diff
@@ -96,11 +96,43 @@
                             "api_interface": "rest",
                             "internal_path": "",
                             "type": "POST",
-                            "add_on": "convex"
+                            "add_on": "compound-v3"
                         },
                         "apis": [
                             {
-                                "name": "/subgraphs/name/convex",
+                                "name": "/subgraphs/name/compound-v3",
```
Comment (Yaroms) [inline cookbook/specs/spec_add_sqdsubgraph.json:103]: is covex not supported anymore?

File: cookbook/specs/spec_add_sqdsubgraph.json [hunk 2]
```diff
@@ -96,11 +96,43 @@
...
+                        "parse_directives": [],
+                        "verifications": []
+                    },
+                    {
+                        "enabled": true,
+                        "collection_data": {
+                            "api_interface": "rest",
+                            "internal_path": "",
+                            "type": "POST",
+                            "add_on": "aave-v3"
```
Comment (Yaroms) [inline cookbook/specs/spec_add_sqdsubgraph.json:131]: do we need 2 different addons for these apis?

File: cookbook/specs/spec_add_sqdsubgraph.json (hunk 1 above)
Comment (sidlavanet) [reply cookbook/specs/spec_add_sqdsubgraph.json:103]: Yes, we are going with Aave and compound for now

File: cookbook/specs/spec_add_sqdsubgraph.json (hunk 2 above)
Comment (sidlavanet) [reply cookbook/specs/spec_add_sqdsubgraph.json:131]: Omer suggested that we add these subgraphs as add-ons. @omerlavanet anything you would like to add here?

Comment (Yaroms) [reply cookbook/specs/spec_add_sqdsubgraph.json:131]: the question is, do node providers need to do something special to support this api? or does all the nodes have them by defualt?
for example trace addon is an additional config for providers to support extra apis than regular node.

File: cookbook/specs/spec_add_sqdsubgraph.json [hunk 3]
```diff
@@ -96,11 +96,43 @@
                             "api_interface": "rest",
                             "internal_path": "",
                             "type": "POST",
-                            "add_on": "convex"
+                            "add_on": "compound-v3"
```
Comment (Yaroms) [inline cookbook/specs/spec_add_sqdsubgraph.json:99]: example:
curl --request POST \
  --url https://sqd-subgraph.net/subgraphs/name/papercliplabs/compound-v3 \
  --header 'Content-Type: application/json' \
  --data '{"query":"{\n  tokens(first: 5) {\n    id\n    name\n    symbol\n    decimals\n  }\n}"}'

**Pull request closed without merging**